### ISSUE TITLE
durable subs client is_offline field seems not to be updated as expected

### ISSUE THREAD

**Issue by chenmarko** [nats-io/nats-streaming-server#556]
Hello,
It seems that although a client disconnects from nat-streaming, the monitor still shows it with:
is_offline: false
Should not it be is_offline: true ?
Is there any graceful timeout setting that I'm missing?

Thanks
Chen

**Comment by kozlovic** [comment]
Are you calling `Close()` on the connection in your client? If not, the server does not know that the client is gone until a certain number of missed heartbeats after which the server will consider the client connection closed. Remember, there is no direct TCP connection between Streaming server and clients. Please have a look at https://github.com/nats-io/nats-streaming-server#relation-to-nats and following chapter.

**Comment by chenmarko** [comment]
Thanks, I've checked that out.
This scenario is of a crashed client. It seems that it takes a good couple of minutes for the server to consider the client connection closed.
Is there a way to expedite this?

**Comment by kozlovic** [comment]
@chenmarko Yes, you can lower the values. Check [here](https://github.com/nats-io/nats-streaming-server#configuring) for the config params (`hb_interval`, `hb_timeout` and `hb_fail_count`). There is also ways to configure them from the command line parameters.

**Comment by chenmarko** [comment]
Thanks, anyway to get the default values for these 3?

**Comment by ripienaar** [comment]
https://github.com/nats-io/nats-streaming-server/blob/0cdba2067b991a346a6da175f82cbbf3a9577a54/server/server.go#L67-L73
